### DISCUSSION TITLE
community shortcut doesnt allow us to add posts

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#20383] [Discussions] [answered]
hey github, i recently found a hidden function on github mobile called "community" from
`settings > send feedback > feature reqest`
and i added community to my shortcuts to open it quickly to see whats going on anytime i want
![IMG_20220712_142343.jpg](https://user-images.githubusercontent.com/94877880/178451853-0202ad4b-aaa2-4cb9-bc10-b25b1e86282d.jpg)

but the community shortcut doesnt allows a user to make posts, if you have to create a new post then you have to again go to `settings > send feedback > feature reqest`
![IMG_20220712_142358.jpg](https://user-images.githubusercontent.com/94877880/178452645-2ca85369-19ee-4f71-8aab-d6a27970cdfd.jpg)

can you add a button on top of community page to creae posts? so a user could make quick discussions? or you can make a fab icon in bottom of the community page to create posts?

![IMG_20220712_142434.jpg](https://user-images.githubusercontent.com/94877880/178452849-274588cf-6783-41e5-85e5-e47d09397417.jpg)
 or add quick community button in top or bottom of the main page of github application?

![IMG_20220712_143429.jpg](https://user-images.githubusercontent.com/94877880/178453851-41cda5b7-b85f-43b4-872d-8a06872f822f.jpg)
or you can add the community icon in top of exolore page or an option "explore community" in explore page?

**Comment by elisealix22** [comment, marked as answer]
@Nit-nit thank you so much for reporting this and for including screenshots! I created an issue internally to track and it seems to be an issue both on iOS and Android specific to discussions shortcuts. In the meantime, a possible workaround might be to add the `github-community/community` repository as a favorite repository on Home and tap through to discussions from there (might be quicker than navigating to Settings). Please let us know if you run into any other issues!
